packages/default/scss/scheduler/_layout.scss

```diff
@@ -811,6 +811,11 @@
             td.k-first {
                 border-left-width: 0;
             }
+
+            // sass-lint:disable no-important
```

Comment:
This will disable sass-lint for the entire file. Use `// sass-lint: disable-line no-important` for the specific line